packages/cli/src/cmds/validator/signers/index.ts

```diff
@@ -3,13 +3,17 @@ import {deriveEth2ValidatorKeys, deriveKeyFromMnemonic} from "@chainsafe/bls-key
 import {interopSecretKey} from "@lodestar/state-transition";
 import {externalSignerGetKeys, Signer, SignerType} from "@lodestar/validator";
 import {toHexString} from "@chainsafe/ssz";
+import {ILogger} from "@lodestar/utils";
 import {defaultNetwork, IGlobalArgs} from "../../../options/index.js";
 import {assertValidPubkeysHex, isValidHttpUrl, parseRange, YargsError} from "../../../util/index.js";
 import {getAccountPaths} from "../paths.js";
 import {IValidatorCliArgs} from "../options.js";
 import {decryptKeystoreDefinitions, PersistedKeysBackend} from "../keymanager/persistedKeys.js";
+import {showProgress} from "../../../util/progress.js";
 import {importKeystoreDefinitionsFromExternalDir, readPassphraseOrPrompt} from "./importExternalKeystores.js";
 
+const KEYSTORE_IMPORT_PROGRESS_MS = 1000;
```

Comment:
I think 1 second may be too fast. Imo 3 or 5 seems better. Thoughts?

Reply:
As @dapplion mentioned the usual rate is 1 key / 7 sec, so I changed the progress time to 10s, so it show some useful progress. 

 https://github.com/ChainSafe/lodestar/pull/5021#discussion_r1080712022